language/stdlib/modules/Event.move

```diff
@@ -69,6 +75,189 @@ module Event {
     public fun destroy_handle<T: copyable>(handle: EventHandle<T>) {
         EventHandle<T> { counter: _, guid: _ } = handle;
     }
+
+    // ****************** SPECIFICATIONS *******************
+
+    /// # Module specifications
+    spec module {
+        /// Helper function that returns whether or not an EventHandleGenerator is
+        /// initilaized at the given address `addr`.
+        define initialized_ehg(addr: address): bool {
```

Comment:
I think we should save "initialized" for things that are called during genesis.  So, perhaps ehg_exists or whatever?